src/ucs/type/status.h

```diff
@@ -70,6 +70,7 @@ typedef enum {
     UCS_ERR_TIMED_OUT              = -20,
     UCS_ERR_EXCEEDS_LIMIT          = -21,
     UCS_ERR_UNSUPPORTED            = -22,
+    UCS_ERR_REMOTE_DISCONNECT      = -24,
```

Comment:
how is this related to the PR? i don't see this error code in the documentation added in this PR..

Reply:
This error code will be returned by ucp_stream_send_nb if the endpoint closed by remote peer. I added this change to this PR because status.h is installable header. Description to ucp_stream_send_nb won't be changed because we don't describe all error codes to API routines.

Reply:
so let's document that closing a stream with close protocol would generate such error on remote side

Reply:
done